build.sbt

```diff
@@ -265,6 +265,21 @@ lazy val `example-todolist-client` = project
   .settings(exampleTodolistCommonSettings)
   .disablePlugins(ScriptedPlugin)
 
+/////////////////////
+//// MARSHALLERS ////
+/////////////////////
+
+lazy val `marshallers-jodatime` = project
+  .in(file("modules/marshallers/jodatime"))
+  .dependsOn(common % "compile->compile;test->test")
```

Comment:
Do we need all those dependencies? I don't see them imported in the code. (Possibly more dependencies are needed for unit tests, but those should be only in test scopes and not exported).

Reply:
I think they will be used in the next PR for adding the marshallers but we could remove them for now 👍